optimize/util/optimize-commons/src/main/java/io/camunda/optimize/upgrade/es/ElasticsearchClientBuilder.java

```diff
@@ -60,10 +67,15 @@ public static String getCurrentESVersion(
     return esClient.withTransportOptions(requestOptions).info().version().number();
   }
 
-  public static RestClient restClient(final ConfigurationService configurationService) {
+  public static RestClient restClient(
+      final ConfigurationService configurationService, final PluginRepository pluginRepository) {
+    final var plugins = extractPluginConfigs(configurationService.getElasticSearchConfiguration());
```

Comment:
Our convention is to not use `var` in the project to my knowledge

Reply:
Sure, will fix that